front/temporal/document_tracker/activities.ts

```diff
@@ -120,3 +122,64 @@ async function getDataSourceDocument({
   }
   return new Ok(docText.value);
 }
+
+/**
+ * Tracker Notification Workflow:
+ * Activity to get tracker ids to notify activity.
+ * They are the active trackers that have generations to consume.
+ * @returns TrackerIdWorkspaceId[]
+ */
+export const getTrackerIdsToNotifyActivity = async (): Promise<
+  TrackerIdWorkspaceId[]
+> => {
+  return TrackerConfigurationResource.internalFetchActivetrackersToProcessByNotificationWorkflow();
+};
+
+/**
+ * Tracker Notification Workflow:
+ * Activity to process tracker notification workflow activity.
+ * Consumes generations for the tracker: fetches the generations, send notifications and consume them.
+ */
+export const processTrackerNotificationWorkflowActivity = async ({
+  trackerId,
+  workspaceId,
+  currentSyncMs,
+}: {
+  trackerId: number;
+  workspaceId: string;
+  currentSyncMs: number;
+}) => {
+  const auth = await Authenticator.internalAdminForWorkspace(workspaceId);
+  const tracker =
+    await TrackerConfigurationResource.fetchWithGenerationsToConsume(
```

Comment:
Will add a check on this to add a filter on the one we need to process regarding the frequency in config!